### PR TITLE
🌐 Add Portuguese Translation for `docs/pt/docs/advanced/custom-response.md`

### PR DESCRIPTION
**Pull request by Joao-Pedro-P-Holanda**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/advanced/custom-response.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
+# Resposta Personalizada - HTML, Stream, File e outras
+
+Por padrão, o **FastAPI** irá retornar respostas utilizando `JSONResponse`.
+
+Mas você pode sobrescrever esse comportamento utilizando `Response` diretamente, como visto em [Retornando uma Resposta Diretamente](response-directly.md){.internal-link target=_blank}.
+
+Mas se você retornar uma `Response` diretamente (ou qualquer subclasse, como `JSONResponse`), os dados não serão convertidos automaticamente (memso que você declare um `response_model`), e a documentação não será gerada automaticamente (por exemplo, incluindo o "media type", no cabeçalho HTTP `Content-Type` como parte do esquema OpenAPI gerado).
```
Comment (ceb10n) [inline docs/pt/docs/advanced/custom-response.md:7]: Just a typo here:

(memso que você declare um `response_model`)

should be

(*me**sm**o* que você declare um `response_model`)

File: docs/pt/docs/advanced/custom-response.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+/// warning | Aviso
+
+Uma `Response` retornada diretamente em sua *função de operação de rota* não será documentada no OpenAPI (por exemplo, o `Content-Type` não será documentado) e não será visível na documentação interativa automática.
+
+///
+
+/// info | Informação
+
+Obviamente, o cabeçalho `Content-Type`, o código de status, etc, irão vir do objeto `Response` que você retornou.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/custom-response.md:92]: I think that "*irão vir*" is a little strange here. Maybe "*virão* do objeto" is better

File: docs/pt/docs/advanced/custom-response.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+
+O **FastAPI** provê a mesma `starlette.responses` como `fastapi.responses` apenas como uma facilidade para você, desenvolvedor. Mas a maioria das respostas disponíveis vêm diretamente do Starlette.
+
+///
+
+### `Response`
+
+A classe principal de respostas, todas as outras respostas herdam dela.
+
+Você pode retorná-la direamente.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/custom-response.md:136]: just a typo: **diretamente**

File: docs/pt/docs/advanced/custom-response.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+
+No exemplo abaixo, o **FastAPI** irá utilizar `ORJSONResponse` por padrão, em todas as *operações de rota*, em vez de `JSONResponse`.
+
+```Python hl_lines="2  4"
+{!../../docs_src/custom_response/tutorial010.py!}
+```
+
+/// tip | Dica
+
+Você ainda substituir `response_class` em *operações de rota* como antes.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/custom-response.md:338]: missed the **pode**:

"*Você ainda **pode** substituir...*"

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great work João 😃 

Just some typos and a few suggestions 😄

**Comment by Joao-Pedro-P-Holanda** [comment]
Fixed @ceb10n, thanks for noticing!

**Review by ceb10n — APPROVED** [review]
❤️

**Comment by alejsdev** [comment]
Thanks for the help! @Joao-Pedro-P-Holanda :rocket: 
And thanks for the reviews @ceb10n @nazarepiedady :mag:

**Pull request merged**